RecoLocalCalo/HGCalRecProducers/interface/DumpClustersDetails.h

```diff
@@ -0,0 +1,186 @@
+#ifndef RecoLocalCalo_HGCalRecProducers_DumpClustersDetails_h
+#define RecoLocalCalo_HGCalRecProducers_DumpClustersDetails_h
+
+#include <vector>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <ctime>
+#include <cstdlib>
```

Comment:
@rovere , header file check failed https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-7ed35b/39812/headers_chks.log 
```
src/RecoLocalCalo/HGCalRecProducers/interface/DumpClustersDetails.h: In member function 'void hgcalUtils::DumpClustersDetails::dumpInfos(const T&, bool) const':
src/RecoLocalCalo/HGCalRecProducers/interface/DumpClustersDetails.h:25:19: error: there are no arguments to 'getpid' that depend on a template parameter, so a declaration of 'getpid' must be available [-fpermissive]
   25 |       pid_t pid = getpid();
```
looks like `unistd.h` is needed here

Reply:
and algorithm as well, I fear.